diff --git a/node_modules/uzip/UZIP.js b/node_modules/uzip/UZIP.js
index 2e30089..7bbe501 100644
--- a/node_modules/uzip/UZIP.js
+++ b/node_modules/uzip/UZIP.js
@@ -157,7 +157,7 @@ UZIP._writeHeader = function(data, o, p, obj, t, roff)
 	wUi(data, o, t==0 ? 0x04034b50 : 0x02014b50);  o+=4; // sign
 	if(t==1) o+=2;  // ver made by
 	wUs(data, o, 20);  o+=2;	// ver
-	wUs(data, o,  0);  o+=2;    // gflip
+	wUs(data, o, 2048);  o+=2;	// gflip
 	wUs(data, o,  obj.cpr?8:0);  o+=2;	// cmpr
 		
 	wUi(data, o,  0);  o+=4;	// time		
